### PR TITLE
Tools: updated year in copyright check

### PR DESCRIPTION
**Pull request by martinfalk**
Fixes outdated year in copyright check.

### CODE REVIEW

**Review by petersteneteg — DISMISSED** [review]

File: modules/python3/processors/MeshCreationTest.py [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# ********************************************************************************
+
 import inviwopy as ivw
-import numpy
+import numpya as np
```
Comment (martinfalk) [inline modules/python3/processors/MeshCreationTest.py]:
```suggestion
import numpy as np
```

File: tools/refactoring/check-copyright.py [hunk 2]
```diff
@@ -1,22 +1,9 @@
 import sys
```
Comment (martinfalk) [inline tools/refactoring/check-copyright.py:1]: copyright?

Comment (martinfalk) [reply tools/refactoring/check-copyright.py:1]: ah, that folder is excluded from the checks.

File: tools/refactoring/check-copyright.py [hunk 3]
```diff
@@ -35,40 +22,11 @@ def print_error(mess, **kwargs):
...
-    years = (currentYear - 1) % 10
-    decades = int((currentYear - 1) / 10) % 10
-    decadesStr = "[0-" + str(decades) + "]" if decades > 0 else "0"
-    yearsStr = "[0-" + str(years) + "]" if years > 0 else "0"
-
-    return re.compile(r"^((?:20" + decadesStr + r"\d-)?20" + decadesStr + yearsStr + r"$)")
-
 
 # match Inviwo Foundation copyright line
 copyrightLine_regex = re.compile(r"\s*[*#]\s+Copyright \(c\) (.*?) Inviwo Foundation\s*")
```
Comment (martinfalk) [inline tools/refactoring/check-copyright.py:27]: So far only the comment indicators ` * ` and ` # ` are considered. Shouldn't `//` be part of that as well?
```suggestion
copyrightLine_regex = re.compile(r"\s*([*#]|//)\s+Copyright \(c\) (.*?) Inviwo Foundation\s*")
```

Comment (petersteneteg) [reply tools/refactoring/check-copyright.py:27]: We don't use that for the copyright block.

**Review by petersteneteg — APPROVED** [review]

**Pull request merged**